screenpipe-server/src/bin/screenpipe-server.rs

```diff
@@ -281,88 +282,93 @@ async fn main() -> anyhow::Result<()> {
     let ocr_engine_clone = cli.ocr_engine.clone();
     let restart_interval = cli.restart_interval;
 
-    // Function to start or restart the recording task
```

Comment:
great!

@joegoldin is there a way to create those runtimes in server/core.rs instead within the start_continuous_recording function? otherwise that's quite redundant/repetitive code

Reply:
Whoops, totally had this in here twice huh?